crates/bevy_math/src/compass.rs

```diff
@@ -111,6 +111,16 @@ impl CompassQuadrant {
         let to_candidate = candidate - origin;
         to_candidate.dot(*dir) > 0.0
     }
+
+    /// Get the dir2 representing this direction
+    pub const fn to_dir2(self) -> Dir2 {
+        match self {
+            CompassQuadrant::North => Dir2::NORTH,
+            CompassQuadrant::East => Dir2::EAST,
+            CompassQuadrant::South => Dir2::SOUTH,
+            CompassQuadrant::West => Dir2::WEST,
+        }
+    }
 }
```

Comment:
These seem fine and could be split off into a separate PR as less controversial.